src/translations.py

```diff
@@ -123,6 +124,8 @@ def _translate(
         num_translated += 1
         try:
             translation = translator.translate(src_strings[k].text, dest=target_lang)
+            if num_translated > 1:
```

Comment:
1. Can you move this after line 139 - `logging.info("Num translated: %d/%d", num_translated, len(src_strings))`?
2. Can you add a small comment before this? Something like `# sleep to avoid hitting Google Translation API's rate limits`?